src/test/java/seedu/address/testutil/EditPersonDescriptorBuilder.java

```diff
@@ -71,6 +73,14 @@ public EditPersonDescriptorBuilder withAddress(String address) {
         return this;
     }
 
+    /**
+     * Sets the {@code Address} of the {@code EditPersonDescriptor} that we are building.
+     */
+    public EditPersonDescriptorBuilder withBirthday(String birthday) {
+        descriptor.setBirthday(new Birthday(birthday));
```

Comment:
Probably can do the `dateUtil` parsing here too

Reply:
The rest of the other 'with' methods doesn't do their parsing in this testutil though. I think what I'm going to do is,

1. Allow birthday to accept a string and parse the string. This is primarily for testing.
2. Create an overloaded constructor which directly accepts a LocalDate.  This is what the app will be using.